### PR TITLE
feat(splitter): add Material skin for Splitter

### PR DESCRIPTION
**Pull request by mvgmvg**
related https://github.com/telerik/kendo-themes/issues/47

### CODE REVIEW

**Comment by mvgmvg** [comment]
As all Material icons uses opacity, I thing that this approach is more consistent with other scenarios with icons. It cannot be mixed, as its parent background (splitbar) has opacity (rgba) as well. Colors used in the mix() function can be only rgb. In this case we are using only 2 splitter variables for Material theme. If we inherit Default theme, we will have four redundant variables.

**Comment by gyoshev** [comment]
Is there a problem with using `rgba()` for the icon colors in order to preserve the opacity?

As for the amount of variables -- they come for free, and allow developers to change the styling of specific interactions with the splitter. You can link the variables that are not part of the default styling.

Reusing existing styles allows us to make changes to one file to fix problems, rather than going through all themes to fix them.

**Comment by mvgmvg** [comment]
You are right, using rgba instead opacity can resolve the issues.

File: packages/material/scss/splitter/_theme.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$splitter-text: rgba($base-text, .54) !default;
+$splitter-bg: rgba($base-text, .12) !default;
+$splitter-hovered-text: $base-text !default;
+$splitter-hovered-bg: rgba($base-text, .22);
+$splitter-selected-text: $selected-text !default;
+$splitter-selected-bg: $selected-bg !default;
+
+@import "~@progress/kendo-theme-default/scss/splitter/theme";
+
+@include exports("splitter/theme") {}
```
Comment (gyoshev) [inline packages/material/scss/splitter/_theme.scss]: This line is not needed, as the module is inherited from the default theme.

**Pull request merged**